### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by Sarahshr**

### CODE REVIEW

File: src/dataProvider.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { fetchUtils } from "react-admin";
+import { api } from "./conf";
+
+const apiUrl = `${api.defaults.baseURL}/admin`;
+const httpClient = fetchUtils.fetchJson;
+
+export default {
+  getList: (resource, params) => {
+    console.log(api.defaults.baseURL);
```
Comment (ThomasMlh) [inline src/dataProvider.js]:
```suggestion
```

**Review by ThomasMlh — COMMENTED** [review]
Please, remove console log

**Review by ThomasMlh — APPROVED** [review]
Ok !

File: src/dataProvider.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { fetchUtils } from "react-admin";
+import { api } from "./conf";
+
+const apiUrl = `${api.defaults.baseURL}/admin`;
+const httpClient = fetchUtils.fetchJson;
+console.log(api.defaults);
```
Comment (LoicBrassart) [inline src/dataProvider.js]: Nope

File: src/dataProvider.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+export default {
+  getList: (resource, params) => {
+    const url = `${apiUrl}/${resource}`;
+
+    return httpClient(url).then(({ headers, json }) => ({
```
Comment (LoicBrassart) [inline src/dataProvider.js:11]: You could have used `axios`, since you import it in this file already

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Once again, a `console.log` !
(Also, enhancements available in `dataProvider` but are not mandatory)

**Review by ThomasMlh — APPROVED** [review]
It's ok !

File: src/dataProvider.js (hunk 1 above)
Comment (LoicBrassart) [reply src/dataProvider.js]: Oh, I'd like you to show how you did this!

**Review by LoicBrassart — APPROVED** [review]
NmE

**Pull request merged**